extensions/wikia/GoogleDocs/GoogleDocs.php

```diff
@@ -11,41 +11,44 @@
  *
  */
 
-if (!defined('MEDIAWIKI')){
-    echo ('THIS IS NOT VALID ENTRY POINT.'); exit (1);
+if ( !defined( 'MEDIAWIKI' ) ) {
+    echo( 'THIS IS NOT VALID ENTRY POINT.' );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the parentheses from this "echo" call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2041)